_people/bart.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Bart de Jonge
+pic: male-no.png
+role: msc
+position: MSc. student
+social:
+- title: twitter
+  url: #
+- title: globe
+  url: https://www.linkedin.com/in/dennis-mouwen-637043123/
```

Comment:
Are you sure you want Dennis' linkedIn profile linked to your name? ;)

Reply:
whoops